index.html

```diff
@@ -6,6 +6,392 @@
+				<p id="WHO">World Health Organization, 2005 <span>3</span></p>
+			</div>
+		</section>
+
+		<!-- I decided to put all the pill bottles into one figure so I could combine them visually with CSS -->
+		<figure id="pillboxes">
+			<figure id="pillb1">
+				<img src="assets/pill-bottle3.svg" alt="red, rectangular pill bottle">
+			</figure>
+			<figure id="pillb2">
+				<img src="assets/pill-bottle.svg" alt="orange, fat pill bottle">
+			</figure>
+			<figure id="pillb3">
+				<img src="assets/pill-bottle2.svg" alt="orange, skinny pill bottle">
+			</figure>
+		</figure>
+	</header>
+	<!-- end intro -->
+
+	<!-- each section past the top header to be identified and provided appropriate whitespace within the class row -->
+
+	<!-- Why people don't adhere -->
+	<section class="row">
+		<span class="subTitle">WHY PEOPLE DON'T ADHERE</span>
+		<h2 class="sectionDescription">A myriad of factors conspire to prevent people from sticking to their treatment programs. But a few are most telling.</h2>
+		<section id="humanBody">
+
+			<!-- The diagram is a series of figures, with respective captions extending into tag bubbles -->
+
+			<figure>
+				<img src="assets/body.svg" alt="a gray outline of a human body">
+				<figure class="tagRight" id="comprehension">
+					<img src="assets/icon-comprehension.svg" alt="Two intertwined gears">
+					<figcaption>
+					<h3>Comprehension</h3>
+					“All of that medical jargon doesn’t make much sense to me but I don’t want to bother the doctor with my silly questions.”
+					</figcaption>
+				</figure>
+				<figure class="tagLeft" id="denial">
+					<img src="assets/icon-denial.svg" alt="A heart symbol">
+					<figcaption>
+					<h3>Denial</h3>
+					“I feel just fine. I don’t know why that doctor thinks I’m sick. Why would I take medicine that I don’t need?”
+					</figcaption>
+				</figure>
+				<figure class="tagRight" id="cost">
+					<img src="assets/icon-cost.svg" alt="A dollar sign">
+					<figcaption>
+					<h3>Cost</h3>
+					“I took my prescription to the pharmacy but they said it would cost several hundred dollars! I just couldn’t afford it.”
+					</figcaption>
+				</figure>
+				<figure class="tagLeft" id="complexity">
+					<img src="assets/icon-complexity.svg" alt="Five tally marks">
+					<figcaption>
+					<h3>Complexity</h3>
+					“Taking 15 medications every day is tough. Especially since they all have different instructions. How am I supposed to keep track of everything?”
+					</figcaption>
+				</figure>
+			</figure>
+		</section>
+	</section>
+	<!-- end why people don't adhere -->
+
+	<!-- The Question -->
+	<section class="row">
+		<span class="subTitle">THE QUESTION</span>
+		<h2 class="sectionDescription">How can technology help patients deal with these factors, so they adhere more accurately and consistently?</h2>
+		<!-- Float both the sub-article and the image left -->
+		<article>
```

Comment:
Agreed. I don't think this should be an `<article>` because the content in here is still related to the heading and text above.  An `<article>`, like a `<section>` implies some kind of change in content/context.